lib/mayaUsd/fileio/primUpdaterManager.cpp

```diff
@@ -582,38 +591,49 @@ PushCustomizeSrc pushExport(const MObject& mayaObject, const UsdMayaPrimUpdaterC
     std::vector<double> timeSamples;
     UsdMayaJobExportArgs::GetDictionaryTimeSamples(userArgs, timeSamples);
 
-    // The pushed Dag node is the root of the export job.
-    std::vector<VtValue> rootPathString(
-        1, VtValue(std::string(dagPath.partialPathName().asChar())));
-    userArgs[UsdMayaJobExportArgsTokens->exportRoots] = rootPathString;
+    const bool isCopy = context.GetArgs()._copyOperation;
+    if (isCopy) {
+        userArgs[UsdMayaJobExportArgsTokens->legacyMaterialScope] = false;
+        userArgs[UsdMayaJobExportArgsTokens->defaultPrim] = "None";
+    } else {
+        // The pushed Dag node is the root of the export job.
+        std::vector<VtValue> rootPathString(
+            1, VtValue(std::string(dagPath.partialPathName().asChar())));
+        userArgs[UsdMayaJobExportArgsTokens->exportRoots] = rootPathString;
+        userArgs[UsdMayaJobExportArgsTokens->legacyMaterialScope] = true;
+    }
 
-    // This ensures the materials will be under the prim, so that
+    // Legacy mode ensures the materials will be under the prim, so that
     // when exported it is under the node being merged and will thus
     // be merged too.
-    userArgs[UsdMayaJobExportArgsTokens->legacyMaterialScope] = true;
 
     UsdMayaJobExportArgs jobArgs = UsdMayaJobExportArgs::CreateFromDictionary(
         userArgs, dagPaths, fullObjectList, timeSamples);
     progressBar.advance();
 
     UsdMaya_WriteJob writeJob(jobArgs);
     if (!writeJob.Write(fileName, false /* append */)) {
-        return pushCustomizeSrc;
+        return result;
     }
     progressBar.advance();
 
-    std::get<SdfPath>(pushCustomizeSrc) = writeJob.MapDagPathToSdfPath(dagPath);
+    result.srcRootPath = writeJob.MapDagPathToSdfPath(dagPath);
+    if (result.srcRootPath.IsEmpty()) {
```

Comment:
When exporting only materials, there might be nothing in the DAG-to-USD map, because the materials are not in the DAG. instead we use the parent scope of the materials, which contains all material when in non-legacy mode. (Duplicate is always in non-legacy mode).